app/src/main/webapp/app/shared/gridresize.ctrl.js

```diff
@@ -0,0 +1,15 @@
+esbMessageAdminApp.controller('gridResizeController', [
```

Comment:
I actually independently arrived at the same conclusion. Will rename it `GridResizeBroadcastController`.